### PR TITLE
Test plans for QUARKUS-2098 — Fix support of media types with suffixes

### PR DESCRIPTION
**Pull request by fedinskiy**

### CODE REVIEW

File: QUARKUS-2098.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- Endpoints with mediatypes with suffix(e.g. application/test+json).
+
+### Impact on test suites and testing automation:
+Negligible. We will add several tests and corresponding endpoints.
+
+## Automated test development
+- Add test for mediatype with subtype and suffix
+- Add tests for mediatype with either subtype or suffix
+- Tests to make sure, that priority is
+  1. git subtype+suffix
```
Comment (mjurc) [inline QUARKUS-2098.md:22]: What does the 'git' mean here?

**Review by mjurc — CHANGES_REQUESTED** [review]
Otherwise, LGTM!

File: QUARKUS-2098.md (hunk 1 above)
Comment (fedinskiy) [reply QUARKUS-2098.md]: Shows importance of not using IDEA's terminal for doing commits. Fixed.

**Review by mjurc — APPROVED** [review]
Thanks for the test plan?

**Pull request merged**